options.yml

```diff
@@ -1,66 +1,216 @@
+        - name: bottomLeft
+          text: '{{window.pos_bottomLeft}}'
+        - name: topRight
+          text: '{{window.pos_topRight}}'
+        - name: bottomRight
+          text: '{{window.pos_bottomRight}}'
+        value: middle
+
+    - url: graphicsApiReplacer.window.continueOutOfFocus
+      header: '{{window.continueOutOfFocus}}'
+      text: '{{window.continueOutOfFocus_descr}}'
+      hasHeader: true
+      display: Choice
+      content:
+        type: choice
+        choices:
+        - name: pause
+          text: '{{window.continueOutOfFocus_pause}}'
+        - name: continue
+          text: '{{window.continueOutOfFocus_continue}}'
+        - name: render
+          text: '{{window.continueOutOfFocus_render}}'
+        value: pause
+
+  - display: GroupBox
+    header: '{{graphic}}'
+    text: '{{graphic_descr}}'
+    hasHeader: true
+    columns: 1
+    children:
+
+    - url: graphicsApiReplacer.graphic.api
+      header: '{{graphic.api}}'
+      text: '{{graphic.api_descr}}'
+      hasHeader: true
+      display: Choice
+      content:
+        type: choice
+        choices:
+        - name: DirectX
+          text: '{{graphic.api_DirectX}}'
+        - name: OpenGL
+          text: '{{graphic.api_OpenGL}}'
+        value: DirectX
+
+    - url: graphicsApiReplacer.graphic.filterLinear
+      header: '{{graphic.filterLinear}}'
+      text: '{{graphic.filterLinear_descr}}'
+      hasHeader: true
+      display: Switch
+      content:
+        type: boolean
+        value: true
+
+    - url: graphicsApiReplacer.graphic.vsync
+      header: '{{graphic.vsync}}'
+      text: '{{graphic.vsync_descr}}'
+      hasHeader: true
+      display: Switch
+      content:
+        type: boolean
+        value: true
+
+    - url: graphicsApiReplacer.graphic.waitWithGLFinish
+      header: '{{graphic.waitWithGLFinish}}'
+      text: '{{graphic.waitWithGLFinish_descr}}'
+      hasHeader: true
+      content:
+        enabled: graphicsApiReplacer.graphic.api == "OpenGL"
+        type: boolean
+        value: false
+
+    - url: graphicsApiReplacer.graphic.pixFormat
+      header: '{{graphic.pixFormat}}'
+      text: '{{graphic.pixFormat_descr}}'
+      hasHeader: true
+      display: Choice
+      content:
+        type: choice
+        choices:
+        - name: argb1555
+          text: '{{graphic.pixFormat_argb1555}}'
+        - name: rgb565
+          text: '{{graphic.pixFormat_rgb565}}'
+        value: argb1555
+
+    - url: graphicsApiReplacer.graphic.debug
+      header: '{{graphic.debug}}'
+      text: '{{graphic.debug_descr}}'
+      hasHeader: true
+      display: Choice
+      content:
+        type: choice
+        choices:
+        - name: none
+          text: '{{graphic.debug_none}}'
+        - name: enabled
+          text: '{{graphic.debug_enabled}}'
+        - name: debugContextEnabled
+          text: '{{graphic.debug_debugContextEnabled}}'
+          enabled: graphicsApiReplacer.graphic.api == "OpenGL"
```

Comment:
I don't support this yet. We could, meaning this option would be greyed out but still show up in the dropdown I guess?

Reply:
Alright, then I will keep it there and will move the other upwards in the hierarchy.

Reply:
Done